src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -688,10 +688,10 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
 				new ContextMenuFlyoutItemViewModel()
 				{
 					Text = "BaseLayoutItemContextFlyoutOpenInNewPane/Text".GetLocalizedResource(),
-					Glyph = "\uE117",
+					Glyph = "\xF117",
```

Comment:
What does it change?

Reply:
The previous glyph had a typo so it didn't load